docs/getting-started-guides/coreos/azure/README.md

```diff
@@ -0,0 +1,187 @@
+# Kubernetes on Azure with CoreOS and [Weave](http://weave.works)
+
+## Introduction
+
+In this guide I will demonstrate how to deploy a Kubernetes cluster to Azure cloud. You will be using CoreOS with Weave, which implements simple and secure networking, in a transparent, yet robust way. The purpose of this guide is to provide an out-of-the-box implementation that can ultimately be taken into production with little change. It will demonstrate how to provision a dedicated Kubernetes master and etcd nodes, and show how to scale the cluster with ease.
+
+## Let's go!
+
+To get started, you need to checkout the code:
+```
+git clone https://github.com/GoogleCloudPlatform/kubernetes
```

Comment:
Just thought this would a good idea at first, but then realised that won't be able to pull-request into a tag... I could put the exact revision, but that's also impossible since revision is unknown until I commit it. In fact, this guide is pointing user to the JavaScript code that is part of the guide and Go code is not compiled as part of this or anything like that. I am considering it to be safe to keep the user on master here, as I am not expecting anyone to touch this Azure provisioning code much at all.